pipeline-steps/templates/jobs/fortify-scan-job.yaml

```diff
@@ -10,13 +10,15 @@ parameters:
   applicationName: ''
   rootFolder: '.'
   qualityGateLevel: 1 # 0 for no quality gate, 1 includes critical issues only, 2 includes high, 3 includes medium and 4 includes low.
-  technologyStack: 'JAVA/J2EE/Kotlin'
-  technologyVersion: '11'
+  technologyStack: 'JS/TS/HTML'
+  technologyVersion: ''
   retryLimit: 20 # 1 minute checks, so limit equals max timeout in minutes.
+  runScan: default
 
 jobs:
   - job: PrepareFortifySecrets
     dependsOn: ${{ parameters.dependsOn }}
+    condition: or(eq('${{ parameters.runScan }}', true), or(eq(variables['Build.SourceBranch'], 'refs/heads/master'), eq(variables['Build.SourceBranch'], 'refs/heads/main'), startsWith(variables['Build.SourceBranch'], 'refs/pull/'))
```

Comment:
Will run the fortify scan if runScan parameter is true, or if the branch is master / main or a PR.